Qualifiers/src/main/res/values-de-sw720dp-2784x1800/strings.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?><!--
+  ~
+  ~  Copyright (c) Microsoft Corporation. All rights reserved.
+  ~  Licensed under the MIT License.
+  ~
+  -->
+
+<resources>
+    <!-- When your app is spanned, Surface Duo will take the resources you have put on this folder
+    (values-sw720dp-2784x1800) for both dual-portrait and dual-landscape, so you can add here specific dimens,
+    strings, colors, etc. for Surface Duo. -->
+    <string name="qualifiers_app_name">Qualifikanten</string>
```

Comment:
It can work, but in other samples we have the English names for the sample-names